redbot/cogs/modlog/modlog.py

```diff
@@ -33,6 +34,18 @@ async def reapply_audittype_migration(self, ctx: commands.Context):
         await modlog.handle_auditype_key()
         await ctx.tick()
 
+    @checks.is_owner()
+    @modlogset.command(name="deletenames")
+    async def modlogset_deletenames(self, ctx: commands.Context):
+        """Delete all stored usernames."""
+        async with modlog._config.custom("CASES").all() as modlog_data:
+            for guild_id, guild_data in modlog_data.items():
+                for case_number, case_data in guild_data.items():
+                    if "last_known_username" in modlog_data[guild_id][case_number]:
+                        del modlog_data[guild_id][case_number]["last_known_username"]
+                    await asyncio.sleep(0)
+        await ctx.tick()
```

Comment:
This `ctx.tick()` is inconsistent with:
- `bankset deletenames`
- `modset deletenames`